client/src/components/TableCells/style.scss

```diff
@@ -0,0 +1,143 @@
+$black: #000;
+$red: #ff0000;
+$light-red: #f13b3b;
+$green: #0c9200;
+$light-green: #4bb941;
+$blue: #0052aa;
+$border: rgb(90, 75, 75);
+$subject-button: #a4ccdf;
+$subject-button-hover: #bfebff;
+
+button {
```

Comment:
It is bad idea to write styles to not unique element, use class for button for example